Index: trunk/Makefile-Linux
===================================================================
--- trunk/Makefile-Linux	(revision 105)
+++ trunk/Makefile-Linux	(working copy)
@@ -21,18 +21,18 @@
 
 jxinit:
 	(cd ${JX_HOME}/include; \
-	 ln -s ../libjcore/code jcore; \
-	 ln -s ../libjx/code jx; \
-	 ln -s ../libjx/image jximage; \
-	 ln -s ../libjx/strings jxstrings)
+	 ln -sf ../libjcore/code jcore; \
+	 ln -sf ../libjx/code jx; \
+	 ln -sf ../libjx/image jximage; \
+	 ln -sf ../libjx/strings jxstrings)
 	(cd ${JX_HOME}/lib; \
-	 ln -s ../ACE/ACE_wrappers/ace/libACE.a libACE-5_4_7.a; \
-	 ln -s ../ACE/ACE_wrappers/ace/libACE.so libACE-5_4_7.so; \
-	 ln -s ../ACE/ACE_wrappers/ace/libACE.so.5.4.7 libACE.so.5.4.7; \
-	 ln -s ../libjcore/libjcore-2_5_0.a libjcore-2_5_0.a; \
-	 ln -s ../libjcore/libjcore-2_5_0.so libjcore-2_5_0.so; \
-	 ln -s ../libjx/libjx-2_5_0.a libjx-2_5_0.a; \
-	 ln -s ../libjx/libjx-2_5_0.so libjx-2_5_0.so)
+	 ln -sf ../ACE/ACE_wrappers/ace/libACE.a libACE-5_4_7.a; \
+	 ln -sf ../ACE/ACE_wrappers/ace/libACE.so libACE-5_4_7.so; \
+	 ln -sf ../ACE/ACE_wrappers/ace/libACE.so.5.4.7 libACE.so.5.4.7; \
+	 ln -sf ../libjcore/libjcore-2_5_0.a libjcore-2_5_0.a; \
+	 ln -sf ../libjcore/libjcore-2_5_0.so libjcore-2_5_0.so; \
+	 ln -sf ../libjx/libjx-2_5_0.a libjx-2_5_0.a; \
+	 ln -sf ../libjx/libjx-2_5_0.so libjx-2_5_0.so)
 	touch ${JX_HOME}/ACE/ACE_wrappers/ace/Svc_Conf_l.cpp
 
 mulberryinit:
